### ISSUE TITLE
Change default datetime as current datetime.

### ISSUE THREAD

**Issue by sagark1510** [xgfe/react-native-datepicker#78]
Hi,

At first when datepicker does not have any date selected at that time how can I change the date and time of picker? Right now it's showing current date and time but I want to give it specific date and time.

**Comment by feyy** [comment]
You can pass the default date with the `date` property.

**Comment by mamartin** [comment]
yes, but in that case placeholder is not shown. Is there a way to to show placeholder before touching the datepicker and then start the datepicker on specific date when opened?

**Comment by assafb81** [comment]
Any updates?

**Comment by iksent** [comment]
We really need an "initialDate" prop.

**Comment by blushi** [comment]
Currently, I am using `hideText` prop and I display date value with a dedicated component (Date picker has absolute position and cover this text component to replicate default date picker behaviour). Then, I just put something like `value || customInitialValue` as `date` prop and bingo.
This could be a workaround for you guys.

**Comment by iksent** [comment]
@blushi, I am having similar workaround, but it is not clean code, we should use special prop for it.

**Comment by eleddie** [comment]
Any updates on this ones?

**Comment by bb-cuongnv** [comment]
+1, solution on this ones ??

**Comment by DavitVosk** [comment]
I could find a solution. I passed extra `initialDate` prop (remember that it should be with date format: something like `moment().startOf('day').toDate()`).  My fork is `https://github.com/GolfPlayed/react-native-datepicker.git`, also I had created a PR here: `https://github.com/xgfe/react-native-datepicker/pull/317`

**Comment by iradkot** [comment]
Any updates?

**Comment by DavitVosk** [comment]
@iradkot did you try my solution?
